index.html

```diff
@@ -235,6 +235,22 @@ <h3 class="separator">Skills</h3>
                     </ul>
                 </div>
             </section>
+            <section class="contact-me">
+                <div>
+                    <h3>Contact me</h3>
+                    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+                        <br>I’d love to help with it!
+                    </p>
+                </div>
+                <form action="https://formspree.io/f/{form_id}" method="post"></form>
```

Comment:
- [ ] please notice you have closed the form tag in this line, you are suppose to put all the form elements, like input fields, textarea and submit button inside the form element for it to work properly. try to fix this :+1: 
- [ ] Also in order to integrate properly with the formspree service you need to create an account there and then you will get a unique id which you are suppose to use in the action attribute replacing the `{form_id}` with your id, please fix it

Reply:
Oh, Thanks for this. very helpful